src/Symfony/Component/PropertyInfo/Util/PhpDocTypeHelper.php

```diff
@@ -49,16 +50,15 @@ public function getTypes(DocType $varType): array
 
         $varTypes = array();
         for ($typeIndex = 0; $varType->has($typeIndex); ++$typeIndex) {
-            $varTypes[] = (string) $varType->get($typeIndex);
-        }
+            $t = $varType->get($typeIndex);
```

Comment:
WDYT of using `$type` instead, it will be better for the readability ?

Reply:
I got the habbit of using one-char variable names for variables having such a small scope in the function. But you're right, ```$type``` would be better.